'ushort' not compiling correctly on mac

**Describe the bug**

@colinleroy  recently added quicktake support doesn't compile on Mac due to unknown type 'ushort'

( which is strange? since type headers should convert 'ushort' into apple's 'unsigned short' )

( not blocking, no urgency. but happened to run across )

**libgphoto2 and gphoto2 version**
latest master (08d0369)

**To Reproduce**

Mac OS Ventura 13.5.2 (22G91)

```
$ gcc --version
Apple clang version 15.0.0 (clang-1500.0.40.1)
Target: arm64-apple-darwin22.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -D_GPHOTO2_INTERNAL_CODE -DLOCALEDIR=\"/usr/local/share/locale\" -DCAMLIBS=\"/usr/local/lib/libgphoto2/2.5.31.1\" -I.. -I.. -I../libgphoto2_port -I/opt/homebrew/include -g -O2 -Werror=unknown-warning-option -Wall -Wextra -Wmost -Werror=incompatible-pointer-types -Werror=implicit -Werror=int-conversion -Wno-error=reserved-identifier -Wno-error=documentation-deprecated-sync -Wno-unused-parameter -c quicktake1x0/qtk-helpers.c  -fno-common -DPIC -o quicktake1x0/.libs/la-qtk-helpers.o
quicktake1x0/qtk-helpers.c:74:59: error: unknown type name 'ushort'; did you mean 'short'?
unsigned char getbithuff (int nbits, unsigned char **raw, ushort *huff)
                                                          ^~~~~~
                                                          short
quicktake1x0/qtk-helpers.c:74:15: error: conflicting types for 'getbithuff'
unsigned char getbithuff (int nbits, unsigned char **raw, ushort *huff)
              ^
quicktake1x0/quicktake1x0.h:75:15: note: previous declaration is here
unsigned char getbithuff (int nbits, unsigned char **raw, unsigned short *huff);
              ^
2 errors generated.
```

( https://github.com/gphoto/libgphoto2/blob/08d03695ead88ceeb0cdf1cc83910bfee80220c8/camlibs/quicktake1x0/qtk-helpers.c#L74 )

--- a/camlibs/digita/serial.c
+++ b/camlibs/digita/serial.c
@@ -29,6 +29,7 @@
 #include <stdlib.h>
 #include <errno.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "libgphoto2/gphoto2-endian.h"
 #include "digita.h"
@@ -336,4 +337,3 @@ int digita_serial_open(CameraPrivateLibrary *dev, Camera *camera)
 
 	return negotiated ? 0 : -1;
 }
-

--- a/camlibs/directory/directory.c
+++ b/camlibs/directory/directory.c
@@ -43,6 +43,7 @@
 # include <sys/mount.h>
 #endif
 #include <fcntl.h>
+#include <unistd.h>
 
 /* will happen only on Win32 */
 #ifndef HAVE_LSTAT

--- a/camlibs/enigma13/enigma13.c
+++ b/camlibs/enigma13/enigma13.c
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2-library.h>
 #include <gphoto2/gphoto2-port-log.h>
@@ -414,8 +415,3 @@ camera_init (Camera *camera, GPContext *context)
 
 
 }
-
-
-
-
-

--- a/camlibs/jd11/serial.c
+++ b/camlibs/jd11/serial.c
@@ -26,6 +26,7 @@
 #include <assert.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2.h>
 #include <gphoto2/gphoto2-port.h>

--- a/camlibs/jl2005c/jl2005c.c
+++ b/camlibs/jl2005c/jl2005c.c
@@ -27,6 +27,7 @@
 #include <fcntl.h>
 #include <string.h>
 #include <math.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2.h>
 #include <gphoto2/gphoto2-port.h>

--- a/camlibs/kodak/dc120/dc120.c
+++ b/camlibs/kodak/dc120/dc120.c
@@ -27,6 +27,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
+#include	<unistd.h>
 
 #include <gphoto2/gphoto2.h>
 #include <gphoto2/gphoto2-port.h>
@@ -434,4 +435,3 @@ int camera_init (Camera *camera, GPContext *context) {
 
         return (GP_OK);
 }
-

--- a/camlibs/kodak/dc120/library.c
+++ b/camlibs/kodak/dc120/library.c
@@ -26,6 +26,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#include <unistd.h>
+
 #include <gphoto2/gphoto2.h>
 
 #include "libgphoto2/i18n.h"

--- a/camlibs/kodak/dc210/library.c
+++ b/camlibs/kodak/dc210/library.c
@@ -23,6 +23,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
+
 #include <gphoto2/gphoto2.h>
 
 #include "libgphoto2/i18n.h"

--- a/camlibs/mars/mars.c
+++ b/camlibs/mars/mars.c
@@ -29,6 +29,7 @@
 #include <fcntl.h>
 #include <string.h>
 #include <math.h>
+#include <unistd.h>
 
 #include <libgphoto2/gamma.h>
 

--- a/camlibs/mustek/io.c
+++ b/camlibs/mustek/io.c
@@ -30,6 +30,7 @@
 
 #include <gphoto2/gphoto2-library.h>
 #include <gphoto2/gphoto2-result.h>
+#include <unistd.h>
 
 
 #include "io.h"

--- a/camlibs/mustek/rs232.c
+++ b/camlibs/mustek/rs232.c
@@ -30,6 +30,7 @@
 #include "io.h"
 
 #include <fcntl.h>
+#include <unistd.h>
 #include <sys/time.h>
 
 

--- a/camlibs/mustek/usb.c
+++ b/camlibs/mustek/usb.c
@@ -27,6 +27,7 @@
 	Implemenation of the USB Version of ExecuteCommand
 */
 #include <string.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2-library.h>
 #include <gphoto2/gphoto2-result.h>

--- a/camlibs/panasonic/coolshot/library.c
+++ b/camlibs/panasonic/coolshot/library.c
@@ -33,6 +33,7 @@
 #include <string.h>
 #include <gphoto2/gphoto2.h>
 #include <time.h>
+#include <unistd.h>
 
 #include "libgphoto2/i18n.h"
 
@@ -726,5 +727,3 @@ int coolshot_build_thumbnail (char *data, int *size)
 
 	return( GP_OK );
 }
-
-

--- a/camlibs/polaroid/pdc320.c
+++ b/camlibs/polaroid/pdc320.c
@@ -36,6 +36,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2-library.h>
 #include <gphoto2/gphoto2-port-log.h>

--- a/camlibs/ptp2/fujiptpip.c
+++ b/camlibs/ptp2/fujiptpip.c
@@ -49,6 +49,7 @@
 #include <string.h>
 #include <stdio.h>
 #include <time.h>
+#include <unistd.h>
 #include <errno.h>
 #include <sys/types.h>
 #ifdef HAVE_SYS_SELECT_H

--- a/camlibs/ptp2/ptpip.c
+++ b/camlibs/ptp2/ptpip.c
@@ -36,6 +36,7 @@
 #include <string.h>
 #include <stdio.h>
 #include <time.h>
+#include <unistd.h>
 #include <errno.h>
 #include <sys/types.h>
 #ifdef HAVE_SYS_SELECT_H

--- a/camlibs/quicktake1x0/qtk-helpers.c
+++ b/camlibs/quicktake1x0/qtk-helpers.c
@@ -71,7 +71,7 @@ int qtk_ppm_size(int width, int height) {
 	return len;
 }
 
-unsigned char getbithuff (int nbits, unsigned char **raw, ushort *huff)
+unsigned char getbithuff (int nbits, unsigned char **raw, unsigned short *huff)
 {
 	static unsigned bitbuf = 0;
 	static int vbits = 0;

--- a/camlibs/quicktake1x0/quicktake1x0.c
+++ b/camlibs/quicktake1x0/quicktake1x0.c
@@ -28,6 +28,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2-library.h>
 

--- a/camlibs/sonydscf55/sony.c
+++ b/camlibs/sonydscf55/sony.c
@@ -25,6 +25,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2.h>
 #include "sony.h"

--- a/camlibs/spca50x/spca50x.c
+++ b/camlibs/spca50x/spca50x.c
@@ -32,6 +32,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2.h>
 #include "libgphoto2/gphoto2-endian.h"

--- a/camlibs/toshiba/pdrm11/pdrm11.c
+++ b/camlibs/toshiba/pdrm11/pdrm11.c
@@ -23,6 +23,7 @@
 
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include <gphoto2/gphoto2.h>
 #include "libgphoto2/gphoto2-endian.h"
@@ -252,4 +253,3 @@ int pdrm11_delete_file(GPPort *port, int picNum)
 
 	return(GP_OK);
 }
-

